tests/sessions_tests/tests.py

```diff
@@ -250,6 +250,19 @@ def test_custom_expiry_datetime(self):
         age = self.session.get_expiry_age(modification=modification)
         self.assertEqual(age, 10)
 
+    def test_non_expiry_datetime(self):
```

Comment:
could you add a docstring to this test? Its purpose isn't entirely clear to me from the name.